service/controller/v5/resource/blobobject/spec.go

```diff
@@ -0,0 +1,13 @@
+package blobobject
+
+const (
+	prefixMaster = "master"
+	prefixWorker = "worker"
+)
+
+type ContainerObjectState struct {
+	ContainerName      string
+	Body               string
+	Key                string
+	StorageAccountName string
+}
```

Comment:
Also I think it would be cooler to have real state type. In this case it would be `type State map[string] ContainerObjectState`.